### PR TITLE
cleanup: add help information for apiserver cel metrics

### LINKED ISSUE #114346: API Server metric compilation_duration_seconds missing help text
Both `compilation_duration_seconds` and `evaluation_duration_seconds` are missing help texts.

https://github.com/kubernetes/kubernetes/blob/09d7286eb570c8ab89c3008efd6afcf590a94a50/staging/src/k8s.io/apiserver/pkg/cel/metrics/metrics.go#L42-L53

### PR DESCRIPTION
**Pull request by qingwave**
Signed-off-by: qingwave <isguory@gmail.com>

#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
Add help information for apiserver cel metrics.

#### Which issue(s) this PR fixes:

Fixes #114346

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: staging/src/k8s.io/apiserver/pkg/cel/metrics/metrics.go [hunk 1]
```diff
@@ -43,12 +43,14 @@ func newCelMetrics() *CelMetrics {
 			Namespace:      namespace,
 			Subsystem:      subsystem,
 			Name:           "compilation_duration_seconds",
+			Help:           "Duration in seconds for compile CEL.",
```
Comment (jiahuif) [inline staging/src/k8s.io/apiserver/pkg/cel/metrics/metrics.go:46]:
```suggestion
			Help:           "CEL compilation time in seconds.",
```
simplify a bit

File: staging/src/k8s.io/apiserver/pkg/cel/metrics/metrics.go [hunk 2]
```diff
@@ -43,12 +43,14 @@ func newCelMetrics() *CelMetrics {
...
 			StabilityLevel: metrics.ALPHA,
 		}),
 		evaluationTime: metrics.NewHistogram(&metrics.HistogramOpts{
 			Namespace:      namespace,
 			Subsystem:      subsystem,
 			Name:           "evaluation_duration_seconds",
+			Help:           "Duration in seconds for evaluate CEL.",
```
Comment (jiahuif) [inline staging/src/k8s.io/apiserver/pkg/cel/metrics/metrics.go:53]:
```suggestion
			Help:           "CEL evaluation time in seconds.",
```

Comment (qingwave) [reply staging/src/k8s.io/apiserver/pkg/cel/metrics/metrics.go]: Thanks, pr updated.

**Pull request merged**